python/grass/jupyter/interactivemap.py

```diff
@@ -347,13 +355,90 @@ def add_layer_control(self, **kwargs):
         else:
             self.layer_control_object = self._ipyleaflet.LayersControl(**kwargs)
 
+    def add_query_button(self):
+        """Add custom features like query button and coordinate retrieval"""
+        import ipywidgets as widgets
+        from IPython.display import HTML
+
+        # A ToggleButton to activate/deactivate query mode
+        query_toggle_button = widgets.ToggleButton(
```

Comment:
Let's use an icon for this, see https://stackoverflow.com/questions/60116974/what-is-the-icon-argument-for-ipywidgets-button
For example, https://fontawesome.com/v4/icon/info this should work

Reply:
I think crosshairs might not communicate well the purpose, I think the info one would be better (if possible).

Reply:
I agree, the crosshairs are used to represent zoom to location.